worlds/aquaria/Regions.py

```diff
@@ -0,0 +1,1377 @@
+        self.world.get_location("King Jellyfish cave, Jellyfish Costume",
+                                     self.player).progress_type = LocationProgressType.EXCLUDED
+        self.world.get_location("Final boss area, bulb in the boss third form room",
+                                     self.player).progress_type = LocationProgressType.EXCLUDED
+        self.world.get_location("Sun Worm path, first cliff bulb", self.player).progress_type = (
+            LocationProgressType.EXCLUDED)
+        self.world.get_location("Sun Worm path, second cliff bulb", self.player).progress_type = (
+            LocationProgressType.EXCLUDED)
+        self.world.get_location("The veil top right area, bulb in the top of the water fall",
+                                     self.player).progress_type = (
+            LocationProgressType.EXCLUDED)
+        self.world.get_location("Bubble cave, bulb in the left cave wall", self.player).progress_type = (
+            LocationProgressType.EXCLUDED)
+        self.world.get_location("Bubble cave, bulb in the right cave wall (behind the ice cristal)",
+                                     self.player).progress_type = (
+            LocationProgressType.EXCLUDED)
+        self.world.get_location("Bubble cave, Verse egg", self.player).progress_type = (
+            LocationProgressType.EXCLUDED)
+        self.world.get_location("Kelp Forest bottom left area, bulb close to the spirit cristals",
+                                     self.player).progress_type = (
+            LocationProgressType.EXCLUDED)
+        self.world.get_location("Kelp forest bottom left area, Walker baby",
+                                     self.player).progress_type = LocationProgressType.EXCLUDED
+        self.world.get_location("Sun temple, Sun key", self.player).progress_type = (
+            LocationProgressType.EXCLUDED)
+        self.world.get_location("The body bottom area, Mutant Costume", self.player).progress_type = (
+            LocationProgressType.EXCLUDED)
+        self.world.get_location("Sun temple, bulb in the hidden room of the right part",
+                                     self.player).progress_type = LocationProgressType.EXCLUDED
+        self.world.get_location("Arnassi ruins, Arnassi Armor",
+                                     self.player).progress_type = LocationProgressType.EXCLUDED
+
+    def adjusting_rules(self, options: AquariaOptions) -> None:
+        """
+        Modify rules for single location or optional rules
+        """
+        self.__adjusting_urns_rules()
+        self.__adjusting_crates_rules()
+        self.__adjusting_soup_rules()
+        self.__adjusting_manual_rules()
+        if options.light_needed_to_get_to_dark_places:
+            self.__adjusting_light_in_dark_place_rules()
+        if options.bind_song_needed_to_get_under_rock_bulb:
+            self.__adjusting_under_rock_location()
+
+        if options.mini_bosses_to_beat.value > 0:
```

Comment:
```suggestion
        if options.mini_bosses_to_beat:
```

I believe this should still work the way you want it to since 0 is falsey in python

Reply:
Yes. But is it clearer? I think that the numeric value is clearer.

Reply:
Fair enough, yeah.